protostar/cheatable_starknet/callable_hint_locals/stop_expect_call_hint_local.py

```diff
@@ -0,0 +1,34 @@
+from typing import Callable
+
+from starknet_py.utils.data_transformer.data_transformer import CairoData
+
+from protostar.cheatable_starknet.controllers.expect_call_controller import (
+    ExpectCallController,
+    ExpectedCall,
+)
+from protostar.cheatable_starknet.callable_hint_locals.callable_hint_local import (
+    CallableHintLocal,
+)
+from protostar.starknet.selector import Selector
+from protostar.starknet import RawAddress, Address
+
+
+class StopExpectCallHintLocal(CallableHintLocal):
+    def __init__(self, controller: ExpectCallController):
+        self._controller = controller
+
+    @property
+    def name(self) -> str:
+        return "stop_expect_call"
```

Comment:
The name of this hintlocal is misleading, since it _asserts_ - not removes the expectation

Reply:
Also, this will be kind of cumbersome to use, since we have to pass all the data again. It'd be better if we returned an `int` (as an assertion `id`) and this hintlocal would act upon it. We would probably need to store things differently, or store additional data because we need to retrieve the expectation later by the returned `id`, but i don't think it'd be an issue to have some wrappers for manipulating contents of this map (please strongly consider using only one map if possible, since synchronizing 2 maps state would be a headache).

Reply:
moreover, above can be implemented in a separate PR, not to stop this one from merging

Reply:
I think we should add this separately

Reply:
@karol-bisztyga please submit an issue to do that. it's a `good-first-issue` as well IMO :)

Reply:
https://github.com/software-mansion/protostar/issues/1509